tests/unittests/OnnxImporterTest.cpp

```diff
@@ -1145,6 +1145,54 @@ TEST(onnx, importSliceNoAxes) {
                   {2, 1, 2, 2} /* output */);
 }
 
+static void importCast(std::string fileName, const char *inputName,
```

Comment:
can you make both params, const ref params?

Reply:
Or `llvm::StringRef`!

Reply:
Done. Nevertheless, note that using `llvm::StringRef` is a bit overkill because for instance, `ONNXModelLoader` wants `inputName` as `const char *`. Then, I need to call `inputName.str().c_str()`.
If we want to use `LLVM::StringRef` everywhere, I think we should start by using `llvm::StringRef` in the internal APIs of Glow :-)